docs/structured-streaming-programming-guide.md

```diff
@@ -2554,11 +2554,11 @@ spark.streams().awaitAnyTermination();   // block until any one of them terminat
 {% highlight python %}
 spark = ...  # spark session
```

Comment:
Honestly, I think we should make this as a self-contained example .. that's maybe the root cause that we have not-working examples. We can do it separately later.